### PR TITLE
Use new syntax

### PR DESCRIPTION
**Pull request by friedger**
This PR
* updates the syntax of the smart contracts to the latest version defined by stacks-blockchain

For testing, I used something like:
```
cargo run --bin clarity-cli check ../clarity-js-sdk/packages/clarity-tutorials/contracts/tokens/non-fungible-token.clar
``` 

Open questions:
* What is the best way to implement the rocket factory? In which contract should the nft be defined? How can I access the nft defined in another contract?

This PR should only be merged when the tests can be run with the new syntax

### CODE REVIEW

**Comment by hstove** [comment]
Thanks @friedger ! I'm going to do some work on top of this, to finish the syntax stuff and fix tests.

File: packages/clarity-tutorials/contracts/rocket-tutorial/rocket-factory.clar [hunk 1]
```diff
@@ -21,45 +21,47 @@
...
 ;; args:
 ;; @user (principal) the principal of the user
 ;; returns: boolean
-(define (can-user-buy (user principal))
+(define-private (can-user-buy (user principal))
   (let ((ordered-at-block
       (get ordered-at-block
-        (fetch-entry orderbook ((buyer user))))))
-    (if (is-none? ordered-at-block) 'true 'false)))
+        (print (map-get? orderbook {buyer (print user)})))))
```
Comment (njordhov) [inline packages/clarity-tutorials/contracts/rocket-tutorial/rocket-factory.clar:50]: Misplaced `print` statements here?

**Comment by friedger** [comment]
Yes, all print calls should go.

I had a discussion whether print should return something meaningful or not.

I read `print` as a function that returns the same value as the argument.
Really easy for debugging.

Terje Norderhaug <notifications@github.com> schrieb am Do., 2. Apr. 2020,
23:05:

**Comment by njordhov** [comment]
> Yes, all print calls should go. I had a discussion whether print should return something meaningful or not. I read `print` as a function that returns the same value as the argument. Really easy for debugging.

Good point that print should return the evaluated argument value. But we can do better. There could be more sophisticated logging facilities, perhaps even with levels like `info` and `debug` that can be disabled in production. Rather than a primitive `print` for debugging, there could be a `spy` form that not only returns the evaluated value, but also outputs the function called with its arguments. Like `(spy (map-get? orderbook {buyer user}))` could output detailed debug info like its location in the code, followed by `map-get?` with the actual evaluated argument values for `orderbook` and `{buyer user}`.

**Comment by friedger** [comment]
superceeded by #63

**Pull request closed without merging**